tests/integration/test_aiohttp.py

```diff
@@ -466,3 +467,20 @@ def test_filter_query_parameters(tmpdir, httpbin):
         cassette_content = f.read()
         assert "password" not in cassette_content
         assert "secret" not in cassette_content
+
+
+@pytest.mark.online
+@pytest.mark.asyncio
```

Comment:
```suggestion

```

Reply:
Why is the mark asyncio is not required ?

Reply:
Sorry i didn't saw the tests run, thanks for the suggestion.